### ISSUE TITLE
[Feature Request] Copy instead of just export.

### ISSUE THREAD

**Issue by juan7732** [0xreeko/gpt2markdown#4]
It'd be nice to copy to clipboard instead of just having it as a download

Something that might look like this:
![image](https://user-images.githubusercontent.com/19902389/219918825-98d8144d-770e-4b82-b20f-ca10b46c1189.png)

**Comment by juan7732** [comment]
Create a separate button, call htmlToMarkdown, and instead of saving, just https://www.w3schools.com/howto/howto_js_copy_clipboard.asp#:~:text=navigator.clipboard.writeText(copyText.value)%3B + a nice little alert or something?

Thoughts?

**Comment by phugadev** [comment]
Yes, I think that would be a great addition! Having a 'copy' option would allow for more flexibility as one may not want to completely export to a file at times.

Do you have any ideas for the alert, or maybe just have a simple animation when the button's clicked on?

**Comment by juan7732** [comment]
No sabia que hablabas español tambien jaja, 

I was thinking I was just thinking a normal js alert with "Markdown has been copied to clipboard".
The animation works too, or even having a button that looks like the copy button instead of what I put on there.
![image](https://user-images.githubusercontent.com/19902389/220112318-26cbaac3-434d-4b10-9b45-6fce99c1928b.png)
